src/votekit/metrics.py

```diff
@@ -0,0 +1,146 @@
+from votekit.cvr_loaders import rank_column_csv
+from votekit.election_types import STV, fractional_transfer
+
+fpath = "/Users/emariedelanuez/VoteKit/src/votekit/mn_clean_ballots.csv"  # path to ballot here
+
+
+def loadin(path):  # load in data
+    minneapolis_data = rank_column_csv(fpath)
+    return minneapolis_data
+
+
+def stv_mini(data):  # get stv ranking list and regular list of candidates
+    testSTV = STV(minneapolis_data, fractional_transfer, seats=3)
+    stv_outcome = testSTV.run_election()
+    race_elected_list = [candidate for candidate in stv_outcome.elected]
+    race_eliminated_list = [candidate for candidate in stv_outcome.eliminated]
+    ranking_list = race_elected_list + race_eliminated_list
+    candidate_list = minneapolis_data.get_candidates()
+
+    return ranking_list, candidate_list
+
+
+def comparision(
+    list1, list2
+):  # compare the intial ranking to whatever order the candidates are listed as
+    n = len(list1)
+    indices = list(range(n))
+    model_dictionary = dict(zip(list1, indices))
+    target_dictionary = {key: model_dictionary[key] for key in list2}
+    unranked_values = list(target_dictionary.values())
+    return unranked_values
+
+
+def perform_bubble_sort(
+    unorderedlist,
+):  # do the actual bubble sort and get the distance
+
```

Comment:
this function should also have a doc string explaining what it does and inputs/outputs